pkg/ddl/ddl_worker.go

```diff
@@ -929,23 +929,6 @@ func writeBinlog(binlogCli *pumpcli.PumpsClient, txn kv.Transaction, job *model.
 	}
 }
 
-// waitDependencyJobFinished waits for the dependency-job to be finished.
```

Comment:
can we remove this unused function? @tangenta

Reply:
This function is not used at the moment, I think we can be deleted.